201-vm-linux-jupyterhub/azuredeploy.json

```diff
@@ -32,7 +32,7 @@
             "type": "string",
             "defaultValue": "CPU-112GB",
             "allowedValues": [
-                "CPU-8GB",
+               "CPU-8GB",
```

Comment:
format the json, e.g. SHIFT+ALT+F in VS Code

Reply:
resolved